lib/Debugger.ProtocolHandler/DebuggerImpl.cpp

```diff
@@ -172,17 +172,24 @@ namespace JsDebug
             return Response::Error(e.what());
         }
 
-        m_breakpointMap.emplace(breakpointId, breakpoint);
+        // Check if the breakpoint exists.  If so, add it to our map and pass back
+        // the breakpoint ID to the client.  Omit the breakpoint ID in the result
+        // message for a pre-existing breakpoint; VS Code depends upon this to
+        // keep in sync with engine breakpoints in this case.
+        if (!ActualBreakpointIdExists(breakpoint))
```

Comment:
What if we instead just returned the ID of the original breakpoint?  Then VS Code could associate the two.